### PR TITLE
Made frontend prettier

### PR DESCRIPTION
**Pull request by eduperezrocha**

### CODE REVIEW

**Review by acescudero — APPROVED** [review]
LGTM. Do you mind taking a look as well @natjungquist ?

File: frontend/src/__tests__/MajorAccordion.test.js [hunk 1]
```diff
@@ -1,35 +1,56 @@
...
+describe('MajorAccordion', function () {
+  function getMockSetSelectedPost () {
+    return jest.fn()
+  }
 
+  it('renders major name with 0 opportunities', function () {
+    render(
+      <MajorAccordion
+        major={mockMajorNoPosts}
+        numPosts={mockMajorNoPosts.posts.length}
```
Comment (natjungquist) [inline frontend/src/__tests__/MajorAccordion.test.js:15]: interesting idea to add numPosts as a prop

File: frontend/src/__tests__/PostList.test.js [hunk 2]
```diff
@@ -1,63 +1,96 @@
...
-    const firstMockProject = mockThreeActiveProjects[0]
-    const firstMockName = `${firstMockProject.name} ${firstMockProject.researchPeriods} ${firstMockProject.faculty.firstName} ${firstMockProject.faculty.lastName} ${firstMockProject.faculty.email}`
-    const tableRow = screen.getByRole('row', { name: firstMockName })
-    fireEvent.click(tableRow)
+    // Use userEvent instead of fireEvent for better interaction simulation
+    await userEvent.click(card)
 
     expect(mockSetSelectedPost).toHaveBeenCalledTimes(1)
-    expect(mockSetSelectedPost).toHaveBeenCalledWith(mockThreeActiveProjects[0])
+    expect(mockSetSelectedPost).toHaveBeenCalledWith(firstProject)
```
Comment (natjungquist) [inline frontend/src/__tests__/PostList.test.js:94]: this is a somewhat cleaner way to test this, good job

File: frontend/src/__tests__/MainAccordion.test.js [hunk 3]
```diff
@@ -1,41 +1,51 @@
...
+      />
+    )
 
-    mockResearchOps.forEach(department => {
-      const departmentHeader = screen.getByText(department.name)
-      expect(departmentHeader).toBeInTheDocument()
+    // Find the major name using test ID instead of text content
+    // to account for multiple elements that include the name of the
+    // major.
+    const majorNameEl = screen.getByTestId('major-name')
```
Comment (natjungquist) [inline frontend/src/__tests__/MainAccordion.test.js:23]: this is better practice, thanks for catching

**Review by natjungquist — APPROVED** [review]
nice job, the app looks way better! thanks for fixing up some of the tests as well to account for the changes

**Comment by eduperezrocha** [comment]
[Code general app theme/styling for UI consistency](https://trello.com/c/EYwIIPLk/25-code-general-app-theme-styling-for-ui-consistency)

**Comment by theonlypal** [comment]
[Code frontend for viewing individual posts](https://trello.com/c/X9MX7mmm/28-code-frontend-for-viewing-individual-posts)

**Pull request merged**